### ISSUE TITLE
The last edited element of a table doesn't save it's changes

### ISSUE THREAD

**Issue by cegredev** [Milchreis/uibooster-for-processing#6]
If you don't "unfocus" (click away from) the last element you edited in a table before hitting okay, it's value doesn't get stored.

For example, run this code...

```
import java.util.*;

void setup() {
  String[][] result = new UiBooster().showTable(new String[][] {}, Arrays.asList("Names"), null);

  System.out.println(Arrays.deepToString(result));
}
```

...add three rows, fill them in, don't unfocus the last element and the output will be:

> [[a], [b], [null]]

**Comment by Milchreis** [comment]
Hi @SchredDev 

thank you for reporting this bug. You are right, the last focused value is not saved on "Ok". I will fix this for the main UiBooster project, for this reason I made a new issue in the main project.

**Comment by Milchreis** [comment]
Fix is deployed with version 1.1.1 🍾 😃
